packages/main/src/themes/Badge.css

```diff
@@ -2,28 +2,49 @@
 
 :host(:not([hidden])) {
 	display: inline-block;
-	height: 1.125em;
+	height: 1em;
```

Comment:
better use rem to be consistent with other components

Reply:
Ems are used here to enable scaling of the icon based on the font size #729